modules/social_features/social_user_export/social_user_export.module

```diff
@@ -85,6 +85,65 @@ function _social_user_export_get_view(array $query = [], $execute = TRUE) {
   return $view;
 }
 
+/**
+ * Returns username of user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return mixed
+ *   The username of the user.
+ */
+function social_user_export_user_name(UserInterface $account) {
+  $query = \Drupal::database()->select('users_field_data', 'ufd');
+  $query->addField('ufd', 'name');
+  $query->condition('ufd.uid', $account->id());
+
+  return $query
+    ->execute()
+    ->fetchField();
+}
+
+/**
+ * Returns first name of user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return string
+ *   First name of the use.
+ */
+function social_user_export_first_name(UserInterface $account) {
+  $query = \Drupal::database()->select('profile__field_profile_first_name', 'pfn');
+  $query->addField('pfn', 'field_profile_first_name_value');
+  $query->join('profile', 'p', 'p.revision_id = pfn.revision_id');
+  $query->condition('p.uid', $account->id());
+
+  return $query
+    ->execute()
+    ->fetchField();
+}
+
+/**
+ * Returns LastName of user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return string
+ *   LastName of the user.
+ */
+function social_user_export_last_name(UserInterface $account) {
+  $query = \Drupal::database()->select('profile__field_profile_last_name', 'pln');
```

Comment:
We should use the Profile entity to retrieve the last name for a user instead of accessing the database directly. See `social_user_user_format_name_alter` for an example.

Also make sure that no errors occur of the user has no profile yet or if the first name field does not exist or has no value (for example due to the privacy module).